imageroot/events/subscription-changed/80sevice_restart

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2023 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+# If the control reaches this step, the service can be enabled and started
+
+systemctl --user try-restart freepbx.service flexisip.service
```

Comment:
```suggestion
systemctl --user try-restart freepbx.service
```
flexisip doesn't need to be rebooted because it calls a [.php](https://github.com/nethesis/ns8-nethvoice/blob/main/freepbx/var/www/html/freepbx/push-proxy.php) that uses credentials and that is in freepbx container

Reply:
flexisip still import the `SUBSCRIPTION_*` variables in the unit file: https://github.com/nethesis/ns8-nethvoice/blob/2140729f782134fb6577efc9e4358abaead4c92b/imageroot/systemd/user/flexisip.service#L24

Reply:
you're right